quickstart-node/src/middlewares/monday-api-error.middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from 'express';
+import { LoggerService } from '@services/monday-code';
+import { HttpStatusCode } from 'axios';
+
+const isMondayApiError = (err: any): boolean => {
```

Comment:
im not sure I like this. the reason is partial errors: you can have an error and the code is still ok. I dont think this need to be a middleware like this but a check made within the query result (unless you are catching something like infra error for instance complexiy but thats not it)